tox.ini

```diff
@@ -42,9 +40,11 @@ commands =
 allowlist_externals =
     make
 
+[pytest]
+addopts = --color=yes --cov=xsdba
```

Comment:
This isn't the best place to put this, since `pytest_ini_options` already exists in `pyproject.toml`. If we want to have colour+coverage always (locally and on CI), we should add this to the `addopts` section in `pyproject.toml`. The reason for keeping it under `testenv.setenv` is so that the options are only specified in a `tox` run.

Reply:
I was just messing to try and make it work. Was the previous implementation better? (still in the same file)

Reply:
before it would have been more like this, and this was not my design 

```python
[testenv]
setenv =
    PYTEST_ADDOPTS = "--color=yes --cov xsdba"
```

Reply:
If not we can change this call, put it in `pyproject.toml`. I'm just a bit confused, I  assume it was OK to put it in `tox.ini` since I don't recall choosing to put this here. It was either you or your cookie-cutter template, something like this.

Reply:
The cookiecutter uses the `PYTEST_ADDOPTS = "--color=yes --cov=xsdba"`, but technically you could place this elsewhere. I wouldn't place a `[pytest]` section here, since this could supersede/collide with the `pytest` args found under `pyproject.toml`. 

Do we *always* want coverage? If so, just put it all under `pyproject.toml` and call it a day.